docs/src/modules/components/LocalizationTable.js

```diff
@@ -18,50 +17,43 @@ const Root = styled('div')(({ theme }) => ({
   borderRadius: 5,
   fontWeight: 600,
   color: (theme.vars || theme).palette.text.secondary,
-
   '&.low': {
     color:
-      (theme.vars || theme).palette.mode === 'dark'
+      theme.palette.mode === 'dark'
         ? (theme.vars || theme).palette.text.primary
         : (theme.vars || theme).palette.error.dark,
-
     '& .progress-bar': {
       backgroundColor: (theme.vars || theme).palette.error.main,
       opacity: 0.3,
     },
     '& .progress-background': {
-      border: `1.5px solid ${(theme.vars || theme).palette.error.dark}`,
+      border: `1px solid ${(theme.vars || theme).palette.error.light}`,
```

Comment:
I love how the lighter versions of the border look on the dark theme. However, I would keep the dark borders on light theme, it looks more consistent with the text. WDYT?
```suggestion
      border: `1px solid`,
      borderColor:
        theme.palette.mode === 'dark'
          ? (theme.vars || theme).palette.error.light
          : (theme.vars || theme).palette.error.dark,
```

Reply:
I have tried to use `.main` but it felt still too much distracting from what I think matters: the % progress.